open-api/typescript-sdk/README.md

```diff
@@ -0,0 +1,26 @@
+# @immich/sdk
+
+A TypeScript SDK for interfacing with the [Immich](https://immich.app/) API.
+
+## Install
+
+```bash
+npm i --save @immich/sdk
+```
+
+## Usage
```

Comment:
Should we also link to the CLI as a more elaborate usage example?

Reply:
Yeah, that's not a bad idea. Let me add a comment.